disnake/channel.py

```diff
@@ -51,7 +51,7 @@
     try_enum,
     VoiceRegion,
     VideoQualityMode,
-    PartyType,
+    ThreadArchiveDuration as ThreadArchiveDurationT,
```

Comment:
since this is an enum and not a typehint, like the conflicting one from .types.threads is, the T is not needed